### PR TITLE
Rebuild node access permissions

### PR DESCRIPTION
**Pull request by MPParsley**

### CODE REVIEW

**Comment by zerolab** [comment]
This can be potentially a very slow process on large sites. Why not call `node_access_needs_rebuild(TRUE)` instead and let the admins trigger the process once the rest of the changes are in?

**Comment by MPParsley** [comment]
> This can be potentially a very slow process on large sites. Why not call `node_access_needs_rebuild(TRUE)` instead and let the admins trigger the process once the rest of the changes are in?

Makes sence, added it.

File: og_access/og_access.install [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * @file
+ * Install/update/uninstall hook implementations.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function og_access_install() {
+  // Rebuild node access permissions.
+  node_access_rebuild(TRUE);
```
Comment (zerolab) [inline og_access/og_access.install]:
```suggestion
  node_access_needs_rebuild(TRUE);
```

Comment (zerolab) [reply og_access/og_access.install]: with a comment of `Mark node access permissions for rebuild.` or something along the lines

**Review by zerolab — CHANGES_REQUESTED** [review]

File: og_access/og_access.install [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+/**
+ * @file
+ * Install/update/uninstall hook implementations.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function og_access_install() {
+  // Rebuild node access permissions.
```
Comment (MPParsley) [inline og_access/og_access.install]:
```suggestion
  // Mark node access permissions for rebuild.
```

File: og_access/og_access.install (hunk 1 above)
Comment (MPParsley) [reply og_access/og_access.install]: whoops :-)

**Pull request merged**